python_tictactoe/app/app_main.py

```diff
@@ -0,0 +1,20 @@
+from python_tictactoe.app.game_config import GameConfig
+from python_tictactoe.ui.cli_messages import CliMessages
+
+class App:
+
+    def run(self):
+        self.game_config(CliMessages)
+
+    def game_config(self, messages):
+        cli_messages = messages()
+        print(cli_messages.welcome_message())
```

Comment:
It would be nice if we had a class here that could stand in for print, but can still handle string input(and maybe it could get passed in on setup). That way we aren't tied to printing to the command line, and it will make it easier to swap out other IO if we need it.